x/sophon/types/keys.go

```diff
@@ -7,4 +7,10 @@ const (
 	StoreKey
 )
 
-var ParamsPrefix = collections.NewPrefix(0)
+var (
+	ParamsPrefix         = collections.NewPrefix(0)
+	SophonIDPrefix       = collections.NewPrefix(1)
```

Comment:
nitpick but Cosmos convention seems to be that standalone keys are named with the suffix "Key" instead of "Prefix." So `ParamsKey` and `SophonIDKey`.